chat-widget/src/common/utils.ts

```diff
@@ -386,3 +393,16 @@ export const getConversationDetailsCall = async (chatSDK: any) => {
 
     return conversationDetails;
 };
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const checkContactIdError = (e: any) => {
+    if (e?.message === "AuthContactIdNotFoundFailure") {
```

Comment:
Can we declare this as constants and use it from there ?

Reply:
It's a `ChatSDKError`. Shall we create a `Enum` for this?